packages/theme/src/colors.ts

```diff
@@ -37,18 +37,21 @@ const light = {
 
     // Figma Colors
     backgroundAlertBlueBold: palette.lightAccentBlue600,
+    backgroundAlertBlueBoldAlt: palette.lightAccentBlue700,
```

Comment:
btw what does the Alt mean?

Reply:
I would say ALTERNATIVE, but dunno, it was in the export.

Reply:
Hey! The "bold-alt" tokens were added as a hotfix to the current token taxonomy due to missing colors for hover states of alert buttons. They intentionally have this generic name because we didn't want to unnecessarily add more new tokens than necessary for expanding alert buttons to new variants (red, yellow, blue). We would have to create a completely separate group with tokens "default" "hover/pressed", or rename the current tokens from "bold" to "default", which, considering the interconnectedness of tokens in various components, didn't seem like the best idea and would add unnecessary complexity. Therefore, it is primarily a temporary solution for the currently set taxonomy and naming, which is largely outdated, and that's why we are working on a new one.

Reply:
This smells to me like temporarily–permanent solution 🫣

Reply:
No way.. work is already underway on the new semantics due to the shortcomings of the current one (issues with accessibility, scalability, and maintenance as our system gradually expands). Unfortunately, it's quite a broad topic that requires extensive analysis to harmonize the requirements for several touchpoints in both design and code simultaneously. Here, we need to address support for multiple platforms simultaneously, while also considering the design aspect and the need to implement a brand facelift with new colors throughout the entire system and so on..

Reply:
Yes, got it :)